### ISSUE TITLE
Repo improvements

### ISSUE THREAD

**Issue by gaieges** [hCaptcha/hmt-basemodels#1]
Jotting down some things here that we should come back to for OSS repo posterity

- [ ] Improved comments that meet our standards in manifest / models content
- [ ] Smaller base docker image (can probably use alpine now that we dont need the web3 stuff)
- [ ] Travis CI autotest and pypi deploy on merge / tag
- [ ] "How to use" documentation for outsiders
- [ ] Tweaks to readme: (no more venv, now pipenv, 
- [ ] Rename python module (`basemodels` -> `hmt-basemodels` or something along those lines) and break out per component (`Manifest`, `RequestConfig`, etc)

**Comment by Villux** [comment]
Should travis handle pypi deploy? Latest version 0.0.6 was not in pypi.

**Comment by Villux** [comment]
Most likely the reason was missing tag 0.0.6

**Comment by gaieges** [comment]
Yep, see here @Villux https://github.com/hCaptcha/hmt-basemodels#note-for-maintainers-deploying-to-pypi

**Comment by Villux** [comment]
hmm no idea anymore on this package. I think we can follow the best practices that eng team has created. I work for ML so I don't know them exactly. Let me know if you need more help
